integration-tests/emulated/chains/parachains/assets/asset-hub-kusama/src/genesis.rs

```diff
@@ -92,6 +103,29 @@ pub fn genesis() -> sp_core::storage::Storage {
 					false,
 					ED,
 				),
+				// Ether
+				(
+					xcm::v4::Location::new(
+						2,
+						[xcm::v4::Junction::GlobalConsensus(EthereumNetworkXcmV4::get())],
+					),
+					AssetHubPolkadotSovereignAccount::get(),
+					true,
+					MIN_ETHER_BALANCE,
+				),
+				// Weth
+				(
+					xcm::v4::Location::new(
+						2,
+						[
+							xcm::v4::Junction::GlobalConsensus(EthereumNetworkXcmV4::get()),
+							xcm::v4::Junction::AccountKey20 { network: None, key: WETH },
+						],
+					),
```

Comment:
Can we put this and `EthereumNetworkXcmV4` in `common.rs` ? I see that it is duplicated here and in `asset-hub-polkadot`

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/625/commits/748af5085e2289813ea502301c4c7173e33633ae.